pytradfri/api/aiocoap_api.py

```diff
@@ -171,10 +171,12 @@ def psk(self, value):
         # Note that this is necessary to start observing
         pr, r = await self._get_response(msg)
 
-        api_command.result = _process_output(r)
+        api_command.result, _ = _process_output(r)
```

Comment:
Same here regarding the checking for a `CHANGED` code, but I think it is unecessary here, because we are just starting the observation.

Reply:
After testing for more than a day, adding a check here would be unnecessary.